packages/backend-defaults/src/entrypoints/scheduler/lib/PluginTaskSchedulerImpl.ts

```diff
@@ -147,6 +152,47 @@ export class PluginTaskSchedulerImpl implements SchedulerService {
     return this.allScheduledTasks;
   }
 
+  getRouter(): express.Router {
+    const router = Router();
+
+    router.get('/.backstage/scheduler/v1/tasks', async (_, res) => {
+      const globalState = await TaskWorker.taskStates(
+        await this.databaseFactory(),
+      );
+
+      const tasks = new Array<TaskApiTasksResponse>();
+      for (const task of this.allScheduledTasks) {
+        tasks.push({
+          taskId: task.id,
+          pluginId: this.pluginId,
+          scope: task.scope,
+          settings: task.settings,
+          taskState:
+            this.localWorkersById.get(task.id)?.taskState() ??
+            globalState.get(task.id) ??
+            null,
+          workerState:
+            this.localWorkersById.get(task.id)?.workerState() ??
+            this.globalWorkersById.get(task.id)?.workerState() ??
+            null,
+        });
+      }
+
+      res.json({ tasks });
+    });
+
+    router.post(
```

Comment:
Thoughts on auth for this one?

Reply:
I think we can keep as is for now seeing as the ability to DoS is kinda limited due to retriggers not affecting current runs